Update _metrics.orderOfOps (#17027)
DataDog/documentation: content/en/dashboards/querying/_index.md

To configure your graph on dashboards, follow this process:

1. [Select the visualization](#select-your-visualization)

> 2. [Define the metric](#Define-the-metric)

Reviewer: ```suggestion
2. [Define the metric](#define-the-metric)
```
Since the `#choose-the-metric-to-graph` heading is changed, the reference links throughout the docs also have to change, for example: https://github.com/DataDog/documentation/blob/aea233bf21d7a2e1391e0b48245588a6761e7d06/content/en/dashboards/guide/quick-graphs.md?plain=1#L69

Let me know if you have questions on this or if you'd like me to commit the reference changes.

3. [Filter your metric](#filter)

> 4. [Configure the time aggregation](#aggregate-and-rollup)

Reviewer: @ijkaylin good catch! I had assumed it was to the parent heading for Aggregate and Rollup, but yes if you want to point to the specific child section, you are correct!
```suggestion
4. [Configure the time aggregation](#configure-your-time-aggregation)
```

> 5. [Configure the space aggregation](#Configure-your-space-aggregation)

Reviewer: ```suggestion
5. [Configure the space aggregation](#configure-your-space-aggregation)
```

Aggregation method is next to the filter dropdown. This defaults to `avg by` but you can change the method to `max by`, `min by`, or `sum by`. In most cases, the metric has many values for each time interval, coming from many hosts or instances. The aggregation method chosen determines how the metrics are aggregated into a single line.

> #### Aggregation groups

Reviewer: Same thing here, reference links will need to be checked and updated.

Next to the aggregation method dropdown, choose what constitutes a line or grouping on a graph. For example, if you choose `host`, there is a line for every `host`. Each line is made up of the selected metric on a particular `host` aggregated using the chosen method.

Additionally, you can click the tags in the metric dropdown used for [choosing the metric](#choose-the-metric-to-graph) to group and aggregate your data.

> #### Rollup to aggregate over time

Reviewer: Reference links will need to be checked and updated

> #### Configure your time aggregation

Reviewer 2: ```suggestion
#### Configure the time aggregation
```

For more about using the JSON view, see [Graphing with JSON][1].

#### Configure your space aggregation

Next to the aggregation method dropdown, choose what constitutes a line or grouping on a graph. For example, if you choose `host`, there is a line for every `host`. Each line is made up of the selected metric on a particular `host` aggregated using the chosen method.

> Additionally, you can click the tags in the metric dropdown used for [defning the metric](#Define-the-metric) to group and aggregate your data. 

Reviewer: ```suggestion
Additionally, you can click the tags in the metric dropdown used for [defining the metric](#define-the-metric) to group and aggregate your data. 
```
